artifactory/commands/jobsummariesimpl/buildpublishjobsummary.go

```diff
@@ -0,0 +1,77 @@
+package jobsummariesimpl
```

Comment:
jobsummariesimpl --> command_output

Let's channge the file name to `command_output.go`